GAE/src/org/akvo/flow/domain/DataUtils.java

```diff
@@ -37,10 +36,9 @@
         if (optionResponse == null || optionResponse.trim() == "") {
             return new String[0];
         }
-
-        List<Map<String, String>> optionNodes = new ArrayList<>();
-        if (optionResponse.startsWith("[")) {
```

Comment:
This original branch structure was to avoid processing a string with the `ObjectMapper` unless it is certain its a JSON string.